### DISCUSSION TITLE
Does anyone have a full working example of running undetected-chromedriver with AWS Lambda?

### DISCUSSION THREAD

**Discussion by emanodame** [ultrafunkamsterdam/undetected-chromedriver#1077] [Q&A]
Title

**Comment by johnebgood** [comment]
A webdriver is a wrapper executable for the locally installed Chrome browser binary. The undetected-chromedriver Python module downloads and patches a local copy of Chrome driver (as undetected_chromedrive.exe on Windows for example) and the locally installed browser is executed on the platform for testing. This is not possible with the default language providers for Lambda. Lambda can be used with custom language providers but then you must implement your own instance/container where it would run. It is possible to run Selenium remote from something like Lambda NodeJS or Python, but then you need to expose ports. Better to run it all in a container from the get go and implement any api interactions to the container.
